bound/kt/src/main/kotlin/web5/sdk/Json.kt

```diff
@@ -0,0 +1,23 @@
+package web5.sdk
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.ObjectWriter
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+
+internal const val dateTimeFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX"
+
+internal object Json {
```

Comment:
need to look at the difference a little more, but a lot of times my ide gives me a yellow suggestion that I should make internal private.

Also should we rename this Web5Json or something, to lower confusion ?

Reply:
I'm open to whatever works best so long as we're not exposing this as a publicly facing api